File: typescipt/tests/parser/variableTypeExtensions.ts


internal static class VariableTypeExtensions {
   public static shouldBePrimitiveType(type: VariableDeclarationType, name: string): void {
     type.ShouldBeOfType<PrimitiveVariableDeclarationType>()
       .Type.ShouldBe(name);
   }
}

File: src/Lexy.Tests/Parser/VariableTypeExtensions.cs
using Lexy.Compiler.Language.VariableTypes;
using Lexy.Compiler.Language.VariableTypes.Declaration;
using Shouldly;

namespace Lexy.Tests.Parser;

internal static class VariableTypeExtensions
{
    public static void ShouldBePrimitiveType(this VariableTypeDeclaration type, string name)
    {
        type.ShouldBeOfType<PrimitiveVariableTypeDeclaration>()
            .Type.ShouldBe(name);
    }
}